techniques/system/distributePolicy/1.0/rudder-rsyslog-relay.st

```diff
@@ -30,7 +30,7 @@ $ActionQueueSaveOnShutdown on
 # If report protocol is HTTPS, we drop the local rsyslog message
 {{#classes.rudder_reporting_https}}
 if $fromhost-ip == "127.0.0.1" then {
-    :programname, isequal, "rudder" ~
```

Comment:
We should maybe support both for compatibility

Reply:
make sense